### ISSUE TITLE
No labels on the wallets checkboxes

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#5788]
**Describe the bug**

Missing labels on the wallets checkboxes.

**To Reproduce**
Steps to reproduce:

1. Go to https://ethereum.org/en/wallets/find-wallet/
2. Scroll down to the "Choose the wallet" section
3. The cards checkboxes don't have labels
![image](https://user-images.githubusercontent.com/468158/160419996-98d8e2fb-9dcf-4dd6-b4ce-308044098cf3.png)

**Expected**

- Each underlying checkbox should have a wrapping `label` element or an `aria-label` attribute.

**Bonus**

- Add styles to highlight the checkbox when they are on focus (probably with `tabindex="0"`).

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by HursitTarcan** [comment]
I can fix this
